.github/workflows/cmake.yml

```diff
@@ -1,4 +1,4 @@
-name: CMake
```

Comment:
What's the rationale for the file name ?

Reply:
Default name for the file.

I changed the pipeline name in the next MR : https://github.com/LouisCharlesC/safe/blob/add-more-test-envs/.github/workflows/cmake.yml

all_platforms.yml ?

Reply:
I was curious, I don't know the convention for these files. Anything is fine with me.